kubernetes/windows/setup.ps1

```diff
@@ -26,7 +26,7 @@ Write-Host ('Creating folder structure')
 Write-Host ('Installing Fluent Bit');
 
     try {
-        $fluentBitUri='https://fluentbit.io/releases/2.0/fluent-bit-2.0.14-win64.zip'
+        $fluentBitUri='https://fluentbit.io/releases/3.0/fluent-bit-3.0.6-win64.zip'
```

Comment:
@pfrcks , 1. can you please attach resource usage with and without this change? 2. and also, did you verify enabling container logs to cover all the supported scenarios, such annotation based filtering, metadata collection and multi-line etc.?

Reply:
will do that using the build that's created on this branch today

Reply:
validated all scenarios mentioned above. also added comparison below